### ISSUE TITLE
Incorrect representation of Circle in utm.yaml

### ISSUE THREAD

**Issue by arrcher** [astm-utm/Protocol#18]
In https://github.com/astm-utm/Protocol/blob/implementation_2020q2/utm.yaml
`geometry` property of `Circle` schema does not look right https://github.com/astm-utm/Protocol/blob/55f03a184d68a7b38b9af39fe397e0a85150df8a/utm.yaml#L367-L368
It should be a ref to already defined Point schema
```
geometry:
  $ref: '#/components/schemas/Point'
```

**Comment by astm-utm** [comment]
Perhaps this was fixed after you opened the issue, but I don't see the problem with the above. Can you please clarify?

**Comment by BenjaminPelletier** [comment]
It's the difference between
```json
"geometry": {
  "type": "Point",
  "coordinates": {
    "type": "Point",
    "coordinates": []
  }
}
```
and
```json
"geometry": {
  "type": "Point",
  "coordinates": []
  }
}
```

**Comment by astm-utm** [comment]
Got it. Fixed in 0.3.4.
